ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,9 @@
+The issues forum is __NOT__ for support requests. It is for bugs and feature requests only.
+Please read https://github.com/angular-ui/bootstrap/blob/master/CONTRIBUTING.md and search
+existing issues (both open and closed) prior to opening any new issue and ensure you follow the instructions therein.
```

Comment:
Right, but no one reads it.  No one sees the yellow banner and the text to click - they are guaranteed to see the text in the box.  I know where you are coming from, but I'm darn near adamant here...please convince me otherwise.